### ISSUE TITLE
How to promote the training effect and convert effect?

### ISSUE THREAD

**Issue by Tian14267** [deepfakes/faceswap#1007]
Hello Author, First, Thank you for great work. I study and run the code those days , but the training effect and convert effect is not good. Just Like this:
![image](https://user-images.githubusercontent.com/27938135/78345041-d4609180-75cf-11ea-8414-63786e286d4e.png)

It is too fuzzy and too unnatural , Even no face outline .
The number of My training dataset is 2000+, and training for 2 days, which just goes 4199 epoch . 
And Loss A: 0.03874, Loss B: 0.04460 . It seems the loss can not be lower.
How can I promote the effect ?

**Comment by torzdf** [comment]
This is best discussed in our Discord (https://discord.gg/FC54sYg) or our Forum (https://forum.faceswap.dev/index.php).

Ultimately it comes down to having better data and more training.

Also see this section in the training guide: https://forum.faceswap.dev/viewtopic.php?f=6&t=146#monitor
